clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -167,7 +167,8 @@ Changes in existing checks
 
 - Improved :doc:`readability-container-contains
   <clang-tidy/checks/readability/container-contains>` check to let it work on
-  any class that has a ``contains`` method.
+  any class that has a ``contains`` method. Also now match previously missing
```

Comment:
I don't think that second sentence is necessary. Original enhancement was introduced in same release.